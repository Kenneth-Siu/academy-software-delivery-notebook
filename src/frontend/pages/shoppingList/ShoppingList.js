import React from "react";
import "./ShoppingList.scss";

export default function ShoppingList() {
    return (
        <main className="shopping-list-page">
            <h1>Shopping List</h1>
        </main>
    );
}
