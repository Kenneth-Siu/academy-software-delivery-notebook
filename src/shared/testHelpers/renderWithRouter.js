import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";

export default function renderWithRouter(component) {
    return render(<MemoryRouter>{component}</MemoryRouter>);
}
