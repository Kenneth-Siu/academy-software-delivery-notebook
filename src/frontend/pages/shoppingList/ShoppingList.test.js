import React from "react";
import renderWithRouter from "../../../shared/testHelpers/renderWithRouter";
import ShoppingList from "./ShoppingList";

describe("ShoppingList", () => {
    test("renders", () => {
        const tree = renderWithRouter(<ShoppingList />);
        expect(tree.container).toMatchSnapshot();
    });
});
