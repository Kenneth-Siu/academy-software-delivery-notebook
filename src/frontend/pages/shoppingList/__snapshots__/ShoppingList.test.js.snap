// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShoppingList renders 1`] = `
<div>
  <main
    class="shopping-list-page"
  >
    <h1>
      Shopping List
    </h1>
  </main>
</div>
`;
